cranelift-wasm/src/sections_translator.rs

```diff
@@ -318,10 +319,7 @@ pub fn parse_element_section<'data>(
                 elems.into_boxed_slice(),
             )?
         } else {
-            return Err(wasm_unsupported!(
-                "unsupported passive elements section: {:?}",
-                kind
-            ));
+            return Err(wasm_unsupported!("unsupported passive elements section",));
```

Comment:
`kind` is no longer `Debug`--should it be?

Reply:
maybe, but it is not useful here